languages/javascript/exercises/concept/arrays-analysis/.meta/config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "contributors": [
```

Comment:
> Also, you're missing a .meta/config.json file with you as author and contributor :)

@SleeplessByte We didn't specify this yet, but our thought was to aim for no duplication here. So if you're an author and a contributor, you're just listed as an author.

Reply:
Oh; at some point the exact opposite was said. I obvi don't care as long as credit is given :)

Reply:
Oh, we'll have to figure this out. Ignore it for now, we can always fix it later if the other option was indeed the correct one :)

CC @iHiD